test/test_helper.rb

```diff
@@ -9,6 +9,7 @@
 require "minitest/autorun"
 require "rgeo-activerecord"
 require "support/fake_record"
+require "active_support/core_ext/date/acts_like"
```

Comment:
why do we need this?

Reply:
AR expects it.  Without it https://github.com/rails/rails/blob/6f57590388ca38ed2b83bc1207a8be13a9ba2aef/activerecord/lib/active_record/sanitization.rb#L228 fails.

Reply:
My concern is that a user loading our gem would have the same issue and need to load this as well. It is in the rails codebase that this should be loaded prior usage, not here in tests. So either our test suite is not initialising properly (I suspect that is the issue) or there is an issue in rails codebase. WDYT?